src/nvim/ex_cmds2.c

```diff
@@ -3324,7 +3324,7 @@ static char_u **find_locales(void)
 {
   garray_T locales_ga;
   char_u      *loc;
-  char *strtok_r_save_ptr = NULL;
+  char *saveptr = NULL;
 
   // Find all available locales by running command "locale -a".  If this
   // doesn't work we won't have completion. */
```

Comment:
I did not see the whole diff, but I assume it also contains `*/`. These should not be here.